### ISSUE TITLE
Working with Trackbar/Slider Control in mobiles (touch) is little hard!

### ISSUE THREAD

**Issue by ahmadshc** [moay/afterglow#74]
My suggestion is to increase the range of touch reception around it to make it easier
Thank you

**Comment by moay** [comment]
We are improving this in v2.

**Comment by ahmadshc** [comment]
> We are improving this in v2.

Thank you very much, I cant find version 2 ? version 2 is under construction ?

**Comment by moay** [comment]
Yes, this is in the makings. Alpha6 is here for first testings: https://github.com/moay/afterglow/releases/tag/2.0.0-alpha6
